### DISCUSSION TITLE
Is there someone who can turn this research into a node?

### DISCUSSION THREAD

**Discussion by Njasa2k** [Comfy-Org/ComfyUI#1325] [General]
https://github.com/naver-ai/DenseDiffusion

Can someone turn this into a node?

**Comment by ntdviet** [comment]
ConditioningArea is already doing exactly that.
I don't understand why they reinvent the wheel

**Reply by Njasa2k** [reply]
It's higher quality. It handles denser captions while following scene layout. I'm not sure why we have to turn any improvement away unless it's reinventing the wheel.

**Reply by ntdviet** [reply]
Is it? I mean "denser captions". The "following scene layout" was never an issue.
Can you show your example? Their demo pic is short of impressive and not particular complex prompts.

Not turning any improvement away, but it needs to show improvements first.

**Reply by ntdviet** [reply]
And btw, that line "I'm not sure why we have to turn any improvement away..." isn't helpful to promote a solution.
I would not be here if I'm not looking for improvements. Your assumption that I'm a conservative throwing rocks at your improvement isn't exactly convincing me to invest time into it.
